device/core/devdoc/device_client_requirements.md

```diff
@@ -159,8 +172,6 @@ The `sendEventBatch` method sends a list of event messages to the IoT Hub as the
 
 **SRS_NODE_DEVICE_CLIENT_16_031: [** The `updateSharedAccessSignature` method shall throw a `ReferenceError` if the sharedAccessSignature parameter is falsy. **]**
 
-**SRS_NODE_DEVICE_CLIENT_06_002: [** The `updateSharedAccessSignature` method shall throw a `ReferenceError` if the client was created using x509. **]**
```

Comment:
>SRS_NODE_DEVICE_CLIENT_06_002 [](start = 2, length = 29)

not sure why you removed this.

Reply:
but I'm sure you had a good reason.  I'm just thinking out loud.

---
In reply to: [157027425](https://github.com/Azure/azure-iot-sdk-node/pull/180#discussion_r157027425) [](ancestors = 157027425)

Reply:
it's another's object responsibility now :)

---
In reply to: [157076486](https://github.com/Azure/azure-iot-sdk-node/pull/180#discussion_r157076486) [](ancestors = 157076486,157027425)